db/migrate/20180201161105_add_verified_attributes_to_identities.rb

```diff
@@ -0,0 +1,12 @@
+class AddVerifiedAttributesToIdentities < ActiveRecord::Migration[5.1]
+  def up
+    add_column :identities, :verified_attributes, :json
+    ActiveRecord::Base.connection.execute(
+      "UPDATE identities SET verified_attributes = '[\"email\"]'"
```

Comment:
I know I suggested this, but after thinking through it I'm realizing this will set the verified attributes when this migration is run, but the code to update this field won't be present until the next release. All the identities created in that time between releases will have `null` here.

With that said, I think we may be able to drop this part of the migration and update this field manually when we deploy this code.